devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy

```diff
@@ -93,7 +93,7 @@ def ping() {
 }
 
 def refresh() {
-    zigbee.onOffRefresh() + zigbee.levelRefresh() + zigbee.onOffConfig() + zigbee.levelConfig()
+    zigbee.onOffRefresh() + zigbee.levelRefresh() + zigbee.onOffConfig(0, 300) + zigbee.levelConfig()
```

Comment:
Yeah this is intentional. Since the cree-bulb actually doesn't do periodic reporting we will rely on the cloud to schedule a healthPoll()